src/microbe_stage/systems/ProcessSystem.cs

```diff
@@ -161,14 +162,16 @@ public sealed class ProcessSystem : AEntitySetSystem<float>
     /// <param name="isPlayerSpecies">Whether this microbe is a member of the player's species</param>
     /// <param name="worldSettings">The world generation settings for this game</param>
     /// <param name="amountType">Specifies how changes during an in-game day are taken into account</param>
+    /// <param name="cache">AutoEvo Cache for speeding up the function</param>
```

Comment:
Auto-evo is written with a hyphen so I'd consider this a typo (also in anywhere else in comments where AutoEvo is written like that without a hyphen).